### PR TITLE
Added resources to build Docker image.

### PR DESCRIPTION
**Pull request by krizsan**
With hope that this shall become the official Elastalert Docker image and make my Elastalert Docker image obsolete.

### CODE REVIEW

**Comment by bean5** [comment]
W00t!

File: Dockerimage/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Elastalert Docker image running on Alpine Linux.
+# Build image with: docker build -t ivankrizsan/elastalert:latest .
```
Comment (bean5) [inline Dockerimage/Dockerfile]: @krizsan you referenced the old repo here.

Comment (krizsan) [reply Dockerimage/Dockerfile]: Sorry, missed that one. Fixed.

**Comment by muravitskiy** [comment]
Usually we don't need supervisord when running single app in container. 
Also you should create container pointing to specific release version, not master branch

**Pull request closed without merging**